dbt_core_integration.py

```diff
@@ -361,6 +361,17 @@ def parse_project(self) -> None:
         self._sql_compiler = None
         self._sql_runner = None
 
+    def apply_defer_config(self, manifest_path: str, favor_state: bool) -> None:
+        with open(manifest_path) as f:
+            manifest = WritableManifest.from_dict(json.load(f))
+            selected = set()
+            self.dbt.merge_from_artifact(
+                self.adapter, other=manifest, selected=selected, favor_state=favor_state
+            )
```

Comment:
I thought we discussed that everytime we need to parse the manifest we need to do the following:

1. parse the manifest as is
2. Write the artifacts
3. merge with defer manifest if defer is on

Reply:
- isn't above the same? the manifest path getting loaded is the deffered manifest and that  is getting merged with current manifest

Reply:
You are right, i missed the call in `rebuildManifest`